source/agora/consensus/protocol/Nominator.d

```diff
@@ -365,10 +373,11 @@ extern(D):
 
     public void stopNominatingTimer () @safe nothrow
     {
-        if (this.nomination_timer !is null)
+        log.info("Stopping nominating timer.");
+        if (this.timers[TimersIdx.Nomination] !is null)
         {
-            this.nomination_timer.stop();
-            this.nomination_timer = null;
+            this.timers[TimersIdx.Nomination].stop();
+            this.timers[TimersIdx.Nomination] = null;
```

Comment:
We actually dont need to null it here, we can reuse the same timer every slot.

Reply:
I did try to remove it but we compare it to null here:
https://github.com/bosagora/agora/blob/83349fdb578db05709b165ca9f045a57f23ba18c/source/agora/consensus/protocol/Nominator.d#L625-L630

I don't think we can check if the timer is `stopped`. 
Maybe `pending` could be used but I think it may be false when actually running the task.

Reply:
then feel free to merge when CI is green

Reply:
Thanks, rebased on a green upstream 🤞